css/css-grid/grid-items/META.yml

```diff
@@ -53,3 +53,101 @@ links:
   results:
   - test: explicitly-sized-grid-item-as-table.html
     status: FAIL
+- product: safari
+  url: https://bugs.webkit.org/show_bug.cgi?id=209461
+  results:
+  - test: grid-item-dynamic-min-contribution-001.html
+    status: FAIL
+  - test: grid-items-percentage-margins-003.html
+    status: FAIL
+  - test: grid-items-percentage-margins-004.html
+    status: FAIL
+  - test: grid-items-percentage-margins-005.html
+    status: FAIL
+  - test: grid-items-percentage-margins-006.html
+    status: FAIL
+  - test: grid-items-percentage-margins-007.html
+    status: FAIL
+  - test: grid-items-percentage-margins-008.html
+    status: FAIL
+  - test: grid-items-percentage-margins-009.html
+    status: FAIL
+  - test: grid-items-percentage-margins-010.html
+    status: FAIL
+  - test: grid-items-percentage-paddings-002.html
+    status: FAIL
+  - test: grid-items-percentage-paddings-005.html
+    status: FAIL
+  - test: grid-items-percentage-paddings-006.html
+    status: FAIL
+  - test: grid-items-percentage-paddings-009.html
+    status: FAIL
+  - test: grid-items-percentage-paddings-010.html
+    status: FAIL
+  - test: grid-items-percentage-paddings-vertical-lr-002.html
+    status: FAIL
+  - test: grid-items-percentage-paddings-vertical-rl-002.html
+    status: FAIL
+- product: safari
+  url: https://bugs.webkit.org/show_bug.cgi?id=209650
```

Comment:
Were you able to root cause the reason for this failure?

Reply:
I have added a new comment to that bug with a more detailed diagnosis of what its contributing to the failures, but I'm still unsure about the final root cause.